core/consensus/grandpa/impl/voting_round_error.cpp

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "consensus/grandpa/impl/voting_round_error.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::consensus::grandpa, VotingRoundError, e) {
+  using E = kagome::consensus::grandpa::VotingRoundError;
+  switch (e) {
+    case E::FIN_VALIDATION_FAILED:
+      return "Validation of finalization message failed";
+    case E::SHOULD_NOT_PRECOMMIT:
+      return "Should not precommit";
```

Comment:
Don't know why I missed it during the previous review, but this message appears not very descriptive to me on what is to be done with this error.